rust/abacus-base/src/oneline_eyre/print.rs

```diff
@@ -0,0 +1,209 @@
+//! This is a port of the backtrace print.rs code which generates one-line backtraces. A lot of
+//! simplifications were made for our specific use case.
+
+use std::ffi::c_void;
```

Comment:
`use std::ffi::c_void;` is not something i ever expected to see in an agent codebase, much less as part of mere debug logging scaffolding